README.md

```diff
@@ -551,7 +551,9 @@ The transcoding process is started by executing the script:
 
 The path is first deleted from the `queue.txt` file and then passed as an argument to the `transcode-video.` tool. To pause after `transcode-video` returns, simply insert a blank line at the top of the `queue.txt` file.
 
-These examples are written in Bash and only supply crop values. But almost any scripting language can be used and any option can be changed on a per input basis.
+These examples are written in Bash and only supply crop values. But almost any scripting language can be used and any option can be changed on a per input basis. A working script with a good progress indiciation is the one written in node and can be found here:
```

Comment:
this is an example of a line-based comment.

Reply:
you could even start a review and "request changes"

Reply:
OK, I'm working on it now but I can guarantee you it will be significantly different. :) I'll start a review when I'm done because I plan to replace the entire patch, not just the paragraph above.

Reply:
no worries ;-)